templates/contributors.html

```diff
@@ -1,40 +1,60 @@
 <!DOCTYPE html>
+{% load staticfiles %}
 <html lang="en">
   <head>
     <!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="{% static 'main.css' %}" rel="stylesheet">
     <title>Contributors Data</title>
   </head>
   <body>
     <h1>Details of all the contributors</h1>
     <ul>
-      {% for contributor in contributors %}
-      <div class="container">
-        <div class="row">
-          <div class="col-sm-6 col-md-4">
-            <div class="thumbnail">
-              <div class="caption">
-                <p>login: {{ contributor.login }}</p>
-                <p>name: {{ contributor.name }}</p>
-                <p>bio: {{ contributor.bio }}</p>
-                <p>num_commits: {{ contributor.num_commits }}</p>
-                <p>reviews: {{ contributor.reviews }}</p>
-                <p>issues_opened: {{ contributor.issues_opened }}</p>
-                <p>teams:
-                {% for team in contributor.teams.all %}
-                    {{ team.name }}
-                {% endfor %}{# for team in contributor.teams.all #}
-                </p>
+      <div class="contributors">
+        {% for contributor in contributors %}
+        <div class="card contributor-card">
+          <div class="empty"></div>
+          <div class="card-action center">
+            <img class="center contributor-image"
+               src="//avatars.githubusercontent.com/{{ contributor.login }}"
+               alt="Profile picture">
+            <h5 class="card-title">
+              {% if contributor.name %}
+                {{ contributor.name }}
+              {% else %}
+                {{ contributor.login }}
+              {% endif %}{# if contributor.name #}
+              <span class="github-icon">
```

Comment:
That's some great work right here!

Just a small suggestion. Currently the usernames are left-shifted a bit to incorporate the `github-icon` in the same line. Maybe you could make the `github-icon` appear in the next row in the center. Apart from this, current design really looks nice.

Reply:
Will make this change soon